include/libtorrent/settings_pack.hpp

```diff
@@ -729,6 +729,8 @@ namespace libtorrent {
 			// ``stop_tracker_timeout`` is the number of seconds to wait when
 			// sending a stopped message before considering a tracker to have
 			// timed out. This is usually shorter, to make the client quit faster.
+			// If the value is set to 0, the connections to trackers with the
+			// stopped event are avoided.
```

Comment:
"avoided" sounds like there's some technique to make it unnecessary to send an ``event=stopped`` announce. What do you think about "suppressed"?